recipes/tensorstore/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "tensorstore" %}
+{% set version = "0.1.44" %}
+# Bazel version is specified upstream, file .bazelversion
+{% set bazel_version = "6.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tensorstore-{{ version }}.tar.gz
+  sha256: 4c95d23c8cfd60ab9e3b109728f49e3b41fa699de33fd54495c0976573cfec27
+  patches:
+    # MacOS needs an extra linker flag so pybind11 finds the python symbols in _tensorstore.so :shrug:
+    - patches/pybind11-linkopts.patch  # [osx]
+
+build:
+  skip: true  # [py<37 or win or osx]
+  number: 0
+  # These libraries should be detected as part of the sysroot but for some reason they aren't?
+  missing_dso_whitelist:       # [linux64]
+   - "*/libpthread.so.0"       # [linux64]
+   - "*/libc.so.6"             # [linux64]
+   - "*/ld-linux-x86-64.so.2"  # [linux64]
```

Comment:
Do we still need these? I see this in the logs:

```
Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_16
Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_16
Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/librt.so.1 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_16
Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/ld-linux-x86-64.so.2 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_16
```

Reply:
Nope. Removed in 3f88bf179e21feb22aa012587b1a6e808a6438f7